packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -1072,8 +1063,10 @@ describe('ReactFlight', () => {
     }
 
     const expectedStack = __DEV__
-      ? // TODO: This should include Throw but it doesn't have a Fiber.
-        '\n    in div' + '\n    in ErrorBoundary (at **)' + '\n    in App'
+      ? '\n    in Throw' +
+        '\n    in div' +
+        '\n    in ErrorBoundary (at **)' +
```

Comment:
The stack we're comparing here is coming from the `componentStack` field passed to Error Boundaries. This will remain the parent stack and available in both DEV and PROD.